variables.tf

```diff
@@ -9,7 +9,7 @@ variable "dns_managed_zone" {
 }
 
 variable "kube_version_gke" {
-  default     = "1.18.17-gke.700"
+  default     = "1.20.9-gke.1001"
```

Comment:
```suggestion
  default     = "v1.18.20-gke.901"
```

Reply:
https://cloud.google.com/kubernetes-engine/docs/release-notes-regular

Reply:
@pgvishnuram [we are not using any release channel](https://github.com/astronomer/google-environments/blob/3af602a0dda66abf4c0f3005c8ad5668379c0b6e/prod/cloud/infra/main.tf#L23). As of [September 7 2021](https://cloud.google.com/kubernetes-engine/docs/release-notes#September_07_2021), version 1.18.20-gke.3001 is still an available control plane version.

Reply:
daniel can we finalise this

Reply:
I talked to @pgvishnuram about this and we need to bump to 1.20 because that is the oldest version supported by the regular channel. These are not the configs that we use internally. We need to either change the channel to UNSPECIFIED, or bump k8s. It seems to me like bumping k8s is the smaller change that would be expected by people who use this, so that's what we're going with. Good catch on those details @pgvishnuram